template/partial/sexual_harassment.html

```diff
@@ -0,0 +1,10 @@
+<p class ="fs17">The Peace Corps is committed to maintaining high standards of conduct in the workplace and providing all employees, Volunteers, and trainees a work environment that is free from harassment of any kind.  The Office of Civil Rights and Diversity, or OCRD, handles claims of harassment, including sexual harassment of a Volunteer or Trainee by a Peace Corps staff member or fellow Volunteer/ Trainee.</p>
+
```

Comment:
Why did you leave empty lines in between? Good coding principles in HTML involve, not leaving lines in between. Is this editor issue, like the ones above?